File: Example_VI_CtCI_Python/Example4.py
def unordPairs2arr(arrayA, arrayB):
    i = 0
    while (i < len(arrayA)):
        j = 0
        while (j < len(arrayB)):
            print(str(arrayA[i]) + "," + str(arrayB[j]))
            j += 1
        i += 1
        
arrayA = [1, 2, 3, 4, 5]
arrayB = [1, 2, 3, 4, 5]
unordPairs2arr(arrayA, arrayB)

File: Examples_VI_CtCI_Java/src/Example4.java

public class Example4 {
	//0 (ab) runtime.
	static void printUnorderedPairs(int[] arrayA, int[] arrayB) {
		for (int i = 0; i < arrayA.length; i++) {
			for (int j = 0; j < arrayB.length; j++) {
				if (arrayA[i] < arrayB[j]) {
					System.out.println(arrayA[i] + ", " + arrayB[j]);
				}
			}
		}
	}
	
	public static void main(String[] args) {
		int[] arrayA = {1, 2, 3, 4, 5};
		int[] arrayB = {1, 2, 3, 4, 5};
		
		printUnorderedPairs(arrayA, arrayB);
	}
}
